### ISSUE TITLE
Change index CSS

### ISSUE THREAD

**Issue by NXPY123** [NXPY123/PageRefresh#3]
Add CSS to index.html

**Comment by yashfalke77** [comment]
@NXPY123 can work on this issue?

**Comment by NXPY123** [comment]
Yeah sure! Just create a CSS file for the index.html file. You can choose the styles you want

**Comment by jaypansuriya104** [comment]
hello I am interested in contribution to this issue ! can you please assign me ?

**Comment by yashfalke77** [comment]
hey @NXPY123 I added css file and added some styles in it, below are the screenshots 
![image](https://user-images.githubusercontent.com/82655175/193292000-0c1ec5d0-03b6-43b7-97b0-ce3b0f20396d.png)

Is there any extra changes needed, if needed then tell me, if not needed can i make the pull request

**Comment by NXPY123** [comment]
@yashfalke77  If you want the PR to be considered for Hacktoberfest, make the PR after October 1(in any time zone, UTC-12 thru UTC+14).

**Comment by yashfalke77** [comment]
@NXPY123  sure, PR requests made from october 2nd will be considered Hackotberfest? and also no changes are needed na?

**Comment by NXPY123** [comment]
@yashfalke77 Yeah PRs from October 2nd will be considered. Yeah the changes are fine 👍

**Comment by yashfalke77** [comment]
hey @NXPY123 I made the pull request , if any changes are neeeded do tag me
